lib/gather-telemetry.js

```diff
@@ -109,8 +109,14 @@ async function buildTelemetry(page) {
           }
         });
 
+        const ownDesc = Object.getOwnPropertyDescriptors(source);
+        const getters = Object.keys(ownDesc).filter(
+          key => ownDesc[key].get && !computedProperties.includes(key)
```

Comment:
Computed Properties also end up as ES5 getters but we exclude them as they are already stored in `computedProperties`